content/en/docs/contribute/review/reviewing-prs.md

```diff
@@ -163,6 +163,17 @@ When reviewing, use the following as a starting point.
 - Do the changes show up in the Netlify preview? Be particularly vigilant about lists, code
   blocks, tables, notes and images.
 
+### Blog
+
+- Early feedback of blog posts are welcome via a Google Doc or HackMD. Please request for early feedback on the [#sig-docs-blog Slack channel](https://kubernetes.slack.com/archives/CJDHVD54J).
+- Before reviewing blog PRs, be familiar with the [Submitting blog posts and case studies](/docs/contribute/new-content/blogs-case-studies/).
+- Blog posts from kubernetes.dev can be mirrored on kubernetes.io within 5 days of initial publication.
```

Comment:
Try this:
```suggestion
- We are willing to mirror any blog article that was published to https://kubernetes.dev/blog/
  (the contributor blog) provided that:
  - the mirrored article has the same publication date
  - for PRs that arrive the original article was merged to https://kubernetes.dev/, there haven't
    been (and won't be) any articles published to the main blog between the original and mirrored
    article publish. This is because we don't want to add articles to people's feeds except at the very end
    of the feed.
  - the original article doesn't contravene any strongly recommended review guidelines or community norms

  You should set the canonical URL for the mirrored article, to the URL of the original article (you can use
  a preview to predict the URL and fill this in ahead of actual publication). Use the `canonicalUrl` field
  in [front matter](https://gohugo.io/content-management/front-matter/) for this.
```

Reply:
I like this update; we privilege timely kubernetes.dev posts, but no other site gets this treatment—all other posts should be original content. Do we want to add anything about specifically not accepting PRs that mirror any other blog's posts? Or is it enough that we say that blog posts should be original content in the [Blogs and Case studies Guidelines section](https://kubernetes.io/docs/contribute/new-content/blogs-case-studies/#guidelines-and-expectations)?

Reply:
I like this update; we privilege timely kubernetes.dev posts, but no other site gets this treatment—all other posts should be original content. Do we want to add anything about specifically not accepting PRs that mirror any other blog's posts? Or is it enough that we say that blog posts should be original content in the [Blogs and Case studies Guidelines section](https://kubernetes.io/docs/contribute/new-content/blogs-case-studies/#guidelines-and-expectations)?

Reply:
We could clarify the special case exceptions (k8s.dev, and also we are willing to mirror the CNCF blog where relevant) that exempt from the original content requirement. I think that's a good point to make.

Reply:
suggestion added